config.ini

```diff
@@ -6,13 +6,14 @@
     password = ""
 [ai]
   # Bootable AI container image to load in your AI partition
-  image = ""
+  image = "quay.io/powercloud/pim:vllm"
   #
   # provide your AI application related configuration inside triple quotes
   config-json = """
   {
-        "llmImage": "<vLLM image>",
-        "llmArgs": "--model ibm-granite/granite-3.2-2b-instruct --max-model-len=52416"
+        "llmImage": "icr.io/ppc64le-oss/vllm-ppc64le:0.10.1.dev852.gee01645db.d20250827",
```

Comment:
Can we leave a comment to specify these configs are meant only for vllm and vllm based apps?